### PR TITLE
Vectra Bug Fix

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc#21623
old PR: https://github.com/demisto/content/pull/6046

## Description
Added Logs outputs in order to get more data about the problem.
Added timestamp validation.

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/Legacy/Integrations/Vectra_v2/Vectra_v2.py [hunk 1]
```diff
@@ -132,19 +132,18 @@ def fetch_incidents(self, last_run: Dict):
         :param last_run: Integration's last run
         """
         last_timestamp: str = last_run.get('last_timestamp', self.first_fetch)  # type: ignore
-
         query_string = f'detection.threat:>={self.t_score_gte}'
         query_string += f' and detection.certainty:>={self.c_score_gte}'
-        query_string += f' and detection.last_timestamp:>{last_timestamp}'
+        query_string += f' and detection.last_timestamp:>{last_timestamp}' #  format: "%Y-%m-%dT%H%M"
```
Comment (ronykoz) [inline Packs/Legacy/Integrations/Vectra_v2/Vectra_v2.py:137]:
```suggestion
        query_string += f' and detection.last_timestamp:>{last_timestamp}'  # format: "%Y-%m-%dT%H%M"
```

File: Packs/Legacy/Integrations/Vectra_v2/Vectra_v2.py [hunk 2]
```diff
@@ -154,14 +153,17 @@ def fetch_incidents(self, last_run: Dict):
             try:
                 for detection in detections:
                     incidents.append(create_incident_from_detection(detection))  # type: ignore
-                    last_timestamp = max_timestamp(last_timestamp, detection.get('last_timestamp'))  # type: ignore
+                    # format from response: %Y-%m-%dT%H:%M:%SZ
+                    response_last_timestamp = datetime.datetime.strptime(detection.get('last_timestamp'),
```
Comment (ronykoz) [inline Packs/Legacy/Integrations/Vectra_v2/Vectra_v2.py:157]: if it doesn't work use parse from dateutil package.

**Comment by Bargenish** [comment]
@ronykoz maybe we should keep the logs (I'll re-write it and make it a bit prettier). We don't have instance for this integration and it could help us with future bugs.

**Comment by ronykoz** [comment]
Sounds good

**Comment by ronykoz** [comment]
@Bargenish lets add changelog and merge this one

**Comment by kirbles19** [comment]
@Bargenish Perfect RN, no change.

**Pull request merged**